### PR TITLE
Iss #214 expand for virtual met mast

### PR DESCRIPTION
**Pull request by stephenholleran**
Please see issue #214 for context.

### CODE REVIEW

**Review by abohara — COMMENTED** [review]
Looks great @stephenholleran . One minor comment but otherwise, looks great.

In `offset_from_utc_hrs`:
Examples show `-5 (for Eastern Standard Time)` -> Should that just be "-5" with the text ?

**Comment by stephenholleran** [comment]
Hi @abohara and @oriollacave,

Please find this pull request with all the things we have discussed over the last few months implemented. It would be great to get your reviews in case I have missed anything?

Once merged, I think this might deserve a new release as this expansion is a big deal.

Thanks both!

**Review by abohara — APPROVED** [review]
@stephenholleran . Looks great. Thanks for all your work on this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,13 +11,19 @@ Additional labels for pre-release and build metadata are available as extensions
 
 ## [Unreleased 2.X.X-2023.XX]
 
-1. To `measurement_type` add:
+1. Adding a new table to capture details of modelled VMM and reanalysis datasets (Issue [#214](https://github.com/IEA-Task-43/digital_wra_data_standard/issues/214))
+   1. This is called `model_config` which is at the same level as `logger_main_config` and contains similar information.
+   1. Resulting duplicate properties `offset_from_utc_hrs`, `averaging_period_minutes` and `timestamp_is_end_of_period` were moved to Definitions.
+   1. `reanalysis` and `virtual_met_mast` enums were added to `measurement_station_type` as outlined below.
```
Comment (oriollacave) [inline CHANGELOG.md:17]: So happy to see this here. Finally. :-)

**Review by oriollacave — APPROVED** [review]
Looks good to me. The title and descriptions are as we have talked and I'm happy with them.
Thanks a lot for the effort @stephenholleran

**Comment by stephenholleran** [comment]
Thanks a million @oriollacave and thanks for your valuable contributions too!

**Pull request merged**